pypa_theme/static/sidebar.js

```diff
@@ -0,0 +1,194 @@
+/*
```

Comment:
Why did we end up with a totally new file here for this? Seems unrelated to the body size change?